src/aws/nodeworker/nodeworker.py

```diff
@@ -53,15 +56,28 @@ async def process(self):
             while True:
                 if not self.current_task and self._task_queue:
                     self.current_task = self._task_queue.pop(0)
+
                     log_info("Downloading File " + self.current_task.file_path + ".")
                     file = self.storage_connector.download_file(
                         file_path=self.current_task.file_path,
                         key=self.current_task.key
                     )
                     log_info("Downloaded file " + self.current_task.file_path + ".")
+                    # file = self.storage_connector.download_file(
+                    #     file_path=self.current_task.file_path,
+                    #     key=self.current_task.key
+                    # )
+                    input_data = self.current_task(0).get_task_data()
```

Comment:
`current_task` is currently a  CommandPacket. How does `current_task(0)` work? And then how does `get_task_data()` are you planning to implement this later on?

Reply:
Oooh this is my misunderstanding. I missed the part where current_task is a command packet. I thought this was a task object which contains the get_task_data() method. I will adjust for this

Reply:
👍